src/ComposerRequireChecker/Cli/CheckCommand.php

```diff
@@ -68,11 +71,21 @@ protected function configure(): void
                 InputOption::VALUE_NONE,
                 'this will cause ComposerRequireChecker to ignore errors when files cannot be parsed, otherwise'
                 . ' errors will be thrown'
+            )
+            ->addOption(
+                'output',
+                null,
+                InputOption::VALUE_REQUIRED,
+                'generate output either as "text" or as "json", if specified, "quiet mode" is implied'
```

Comment:
Is it possible to define a value range upfront? Wondering if the symfony console component can do that.

Reply:
It's not directly possible. But I'll implement the `protected function initialize(...): void` method in the CheckCommand in which the option value will be validated.